server

```diff
@@ -174,6 +460,11 @@ main() {
         SCYLLA_VERSION="$2"
         shift 2
         ;;
+      "--list-active-releases")
+        check_product
```

Comment:
why you are checking product in this option?
are you planning to use output of this function for `system="scylla"` line 275 instead of hardcode it?

Reply:
Actually it's not related to any product, just a general command , so i will remove it